build/configs/artik05x/artik05x_download.sh

```diff
@@ -273,25 +272,40 @@ while test $# -gt 0; do
 		--verify)
 			VERIFY=verify
 			;;
-		ALL|OS|APPS|ROM|BL1|BL2|SSSFW|WLANFW|OTA|MICOM|WIFI|USER|all|os|apps|rom|bl1|bl2|sssfw|wlanfw|ota|micom|wifi|user)
-			download $1
-			;;
-		USBrule|usbrule)
-			RET=$(${THIS_PATH}/../usbrule.sh artik 0403 6010)
-			if [ -n "${RET}" ]; then
-				echo $RET
-				exit 1
-			fi
-			echo "USB rule creation succeeded."
-			exit 0
-			;;
-		ERASE_*)
-			erase
-			;;
 		*)
-			usage 1>&2
-			exit 1
+			break
 			;;
 	esac
 	shift
 done
+
+case $1 in
+	ALL|OS|APPS|ROM|BL1|BL2|SSSFW|WLANFW|OTA|MICOM|WIFI|USER|all|os|apps|rom|bl1|bl2|sssfw|wlanfw|ota|micom|wifi|user)
```

Comment:
How about not distinguising the upper and lower case?

Reply:
I agree with your opinion.
I will change this.

Reply:
modified